### PR TITLE
BTAT-9987 update error message

### PR DESCRIPTION
**Pull request by SheilaTran**

### CODE REVIEW

**Comment by SheilaTran** [comment]
**Please merge with: https://github.com/hmrc/change-vat-details-acceptance-tests/pull/191**

File: app/config/frontendAppConfig.scala [hunk 1]
```diff
@@ -152,6 +154,9 @@ class FrontendAppConfig @Inject()(val runModeConfiguration: Configuration, envir
   override val agentServicesHost: String = sc.getString(ConfigKeys.agentServicesHost)
   override val agentServicesUrl: String = agentServicesHost + sc.getString(ConfigKeys.agentServicesUrl)
 
+  override val agentClientServicesHost: String = sc.getString(ConfigKeys.agentClientHost)
+  override val agentClientServicesUrl: String = agentClientServicesHost + sc.getString(ConfigKeys.agentClientServicesUrl)
```
Comment (lewishay) [inline app/config/frontendAppConfig.scala:158]: Can you remove lines 75 and 157 and simply update your URL value to be `sc.getString(host) + sc.getString(url)`? Looks unnecessary to me to make the host accessible like this when it's not being reused for any other purposes.

File: conf/application.conf [hunk 2]
```diff
@@ -217,6 +217,11 @@ agent-services {
   url = "/agent-services-account"
 }
 
+agent-client-services {
+   host = "http://localhost:9448"
```
Comment (lewishay) [inline conf/application.conf:221]: You will need to make an `app-config-base` PR to blank out this host in the environments

**Review by lewishay — CHANGES_REQUESTED** [review]

**Comment by SheilaTran** [comment]
App-config-base PR: https://github.com/hmrc/app-config-base/pull/6528

**Pull request merged**